### PR TITLE
Fixed typo, see above code

### PR DESCRIPTION
**Pull request by dvdsndr**

### CODE REVIEW

File: app/code/core/Mage/Paypal/Model/Hostedpro/Request.php [hunk 1]
```diff
@@ -157,7 +157,7 @@ protected function _getOrderData(Mage_Sales_Model_Order $order)
     {
         $request = array(
             'subtotal'      => $this->_formatPrice($order->getBaseSubtotal()),
-            'tax'           => $this->_formatPrice($order->getBaseTaxAmount()),
+            'tax'           => $this->_formatPrice($order->getBaseTaxAmount() + $order->getHiddenTaxAmount() ),
```
Comment (LeeSaferite) [inline app/code/core/Mage/Paypal/Model/Hostedpro/Request.php]: Can you explain the reason to add this here?

File: app/code/core/Mage/Catalog/Model/Convert/Parser/Product.php [hunk 2]
```diff
@@ -507,7 +507,7 @@ public function unparse()
 
             $baseRowData = array(
                 'store'     => $row['store'],
-                'website'   => $row['website'],
+                'website'   => $row['websites'],
```
Comment (LeeSaferite) [inline app/code/core/Mage/Catalog/Model/Convert/Parser/Product.php]: Good find!

One question though, does the $baseRowData key need to change as well?

**Comment by LeeSaferite** [comment]
TBH, I'd rather this be two PRs, even if they are each a single line.

File: app/code/core/Mage/Paypal/Model/Hostedpro/Request.php (hunk 1 above)
Comment (dvdsndr) [inline app/code/core/Mage/Paypal/Model/Hostedpro/Request.php]: customer places an order with discount which generates hidden tax in magento. PayPal does its own discount and tax calculation, without the mod paypal marks the order as fraud as the magneto and paypal totals dont match. Others have found the same problem

https://magento.stackexchange.com/questions/49464/magento-1-9-1-bug-discount-tax-paypal-pro-incorrect-total

**Comment by dvdsndr** [comment]
Sorry about the combo PR, i'm new to github processes, how do I submit one then a next, GitHub forced me down the merge path??

File: app/code/core/Mage/Catalog/Model/Convert/Parser/Product.php (hunk 2 above)
Comment (dvdsndr) [inline app/code/core/Mage/Catalog/Model/Convert/Parser/Product.php]: Hmm, cant recall, but what ever problem we had went away with this mod only

**Comment by colinmollenhour** [comment]
@dvdsndr You can reset your commit without losing your changes and then push them to separate branches for separate PRs. There are a hundred ways to do this but here is one:

    $ git fetch   # get up-to-date with the latest upstream
    $ git checkout -b bugfix-1    # create a new branch with all PR changes
    $ git checkout -b bugfix-2    # create a second new branch with all PR changes
    $ git rebase -i origin/1.9.3.x   # in the editor delete all commits that don't pertain to "bugfix-2"
    $ git push origin bugfix-2
    $ git checkout bugfix-1
    $ git rebase -i origin/1.9.3.x  # in the editor delete all commits that don't pertain to "bugfix-1"
    $ git push origin bugfix-1

**Comment by seansan** [comment]
+1 maybe someone else can suggest the merge? or split? I agree that when working on website version youo are quite limited ;)

**Comment by dvdsndr** [comment]
Good idea @seansan, got lost trying @colinmollenhour suggestion.

**Comment by colinmollenhour** [comment]
Merged via command line.

**Comment by LeeSaferite** [comment]
@colinmollenhour Did you merge this code? It was meant to be split into two PRs so we could review and approve each one individually.

**Comment by colinmollenhour** [comment]
Yes. I found the parser fix to be an obvious bug and considering the PayPal fix was well-documented on MSE I decided to go ahead and commit it as well.

**Pull request closed without merging**